support/client/lib/vwf/model/jPlayer.js

```diff
@@ -342,6 +360,13 @@ define( [
 
     } );
 
+    function videoEndedCallback(){
+        var mediaManagerID = vwf.find( undefined, "/mediaManager" )[ 0 ];
+        var videoManagerID = vwf.find( mediaManagerID, "videoManager" ) [ 0 ];
+        console.log("Video ended callback in driver fired!");
+        vwf.fireEvent(videoManagerID, "videoEnded");
```

Comment:
If it's not appropriate, what's the best alternative?  @BrettASwift @davideaster @eric79